docs/spec/spec.md

```diff
@@ -2487,7 +2487,8 @@ http:Client c = check new ("https://localhost:9090",
 );
 
 public function main() returns error? {
-    json response = check c->get("/foo/bar");
+    http:Request req = new;
+    json response = check c->post("/foo/bar", req);
```

Comment:
Shall we get rid of request and add simple json payload inline?

Reply:
Since, we hv used `http:Request` in all security code sample we will use the same approach for consistency.